components/image/image.html

```diff
@@ -53,50 +51,61 @@ <h3 class="sidebar-heading">Components</h3>
-                        <img src="/assets/card-image-2.jpg" alt="image"/>
+            </aside>
+            <main class="main">
+                <div class="component_wrapper">
+                    <h2 class="component_heading">Image</h2>
+                    <p>
+                        The image component has basic 2 types,
+                        <span class="component_description">Responsive Image</span> and 
+                        <span class="component_description">Round Image</span> 
+                        
+                    </p>
+                    <h3 class="component_heading">Responsive Image</h3>
+                    <div class="component_wrapper_box flex-center">
+                        <div class="responsive-image-container">
+                            <img  class="responsive-image" src="/assets/avatar_sample.jpg" alt="image"/>
+                        </div>
                     </div>
-                </div>
-
-                <div class="component_iframe">
-                    <iframe
-                        src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=htmlmixed&width=800&ds=false&dsyoff=0px&dsblur=100px&wc=true&wa=false&pv=0px&ph=0px&ln=true&fl=1&fm=Hack&fs=13.5px&lh=181%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522responsive-image%2522%253E%253Cimg%2520src%253D%2522...%2522%2520alt%253D%2522...%2522%252F%253E%253C%252Fdiv%253E"
-                        class="image-iframe"
-                        sandbox="allow-scripts allow-same-origin">
-                    </iframe>
-                </div>
-
-                <h3 class="component_heading">Round Image</h3>
-                <div class="component_wrapper_box">
-                    <div class="flex-center">
-                        <img class="round-image" src="/assets/card-image-2.jpg" alt="image"/>
+    
+                    <div class="component_iframe">
+                        <iframe
+                            src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=htmlmixed&width=800&ds=false&dsyoff=0px&dsblur=100px&wc=true&wa=false&pv=0px&ph=0px&ln=true&fl=1&fm=Hack&fs=13.5px&lh=181%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522responsive-image%2522%253E%253Cimg%2520src%253D%2522...%2522%2520alt%253D%2522...%2522%252F%253E%253C%252Fdiv%253E"
+                            class="image-iframe"
+                            sandbox="allow-scripts allow-same-origin">
+                        </iframe>
+                    </div>
+    
+                    <h3 class="component_heading">Round Image</h3>
+                    <div class="component_wrapper_box">
+                        <div class="flex flex-center">
+                            <img class="round-image" src="/assets/avatar_sample.jpg" alt="image"/>
+                        </div>
+                    </div>
+    
+                    <div class="component_iframe">
+                        <iframe
```

Comment:
Because you are using carbon.sh you can maybe add loading="lazy" and also if you want to make it responsive you can add width:100%.Just a suggestion.